### DISCUSSION TITLE
add textfield on each pane for file filtering

### DISCUSSION THREAD

**Discussion by stccorp** [doublecmd/doublecmd#1394] [Suggestions]
Hello, I use a program called free commander on windows, They have a textfiled on each pane where you can type and it will instantly filter the files on the pane.  I am new to doublecmd so not sure if there is a way to do this already.

If this option is not there , could it be something that be added to the application?

Thank you

**Comment by ags1234** [comment]
Off topic: I also use FreeCommander. It is the slowest orthodox file manager that currently exists. But it has a lot of advanced options, many of them very interesting and useful.

**Comment by Skif-off** [comment]
[Quick search/filter](https://doublecmd.github.io/doc/en/help.html#cm_QuickSearch)?

**Comment by stccorp** [comment]
Thank you.  thats what I was looking for. its interesting that the program closes the quick search portion when you move to another part of the program instead of being there all the time after using it. I would think that since it was opened by the user, it should only be closed by the user and not automatically.

**Reply by Skif-off** [reply]
= I would think that since it was opened by the user, it should only be closed by the user and not automatically.

I'm not sure I agree with this idea :)
Go to "Configuration" in the main menu > Options... > [Quick search/filter](https://doublecmd.github.io/doc/en/configuration.html#ConfigQuick): _Hide filter panel when not focused_.

**Comment by stccorp** [comment]
Thank you.
 I unchecked the 'Hide filter panel when not focused' and click Apply, But the panel still hides automatically. I am using doublecmd-gtk-latest-x86_64.AppImage on ubuntu 22
